### PR TITLE
Fix permission bug

### PR DESCRIPTION
**Pull request by kamilsa**
Signed-off-by: kamilsa <kamilsa16@gmail.com>

### Description of the Change

Fixes problem of missed grantable permissions in proto file. 

Bug example:
User "userone@domain" wants to grant user "usertwo@domain" to add key value information. To do so "userone@domaiun" needed to grant "can_set_my_account_detail" permission to "usertwo@permission". But to do so user "userone@domain" needs permission "can_grant_can_set_my_quorum" which was missing before in proto files although existed in iroha.

### Benefits

Includes missing permissions to proto file.

### Possible Drawbacks 

None

### Usage Examples or Tests

tx-example.py has been updated to demonstrate granting permission to set account detail to another user, set it, and check if detail has been set.

### CODE REVIEW

File: example/python/tx-example.py [hunk 1]
```diff
@@ -188,11 +210,26 @@ def get_account_asset():
 
     print(query_response)
 
+def get_user_detail():
+    # get detail of the user
```
Comment (nickaleks) [inline example/python/tx-example.py]: Not very informative comment :)

File: example/python/tx-example.py [hunk 2]
```diff
@@ -188,11 +210,26 @@ def get_account_asset():
...
+    query = query_builder.creatorAccountId(creator) \
+        .createdTime(current_time) \
+        .queryCounter(11) \
```
Comment (nickaleks) [inline example/python/tx-example.py]: Why is it 11?

Comment (muratovv) [reply example/python/tx-example.py]: If you want to get unique query counter, maybe use a static variable with increment instead?

File: example/python/tx-example.py [hunk 3]
```diff
@@ -156,6 +156,28 @@ def tx4():
     send_tx(tx, key_pair)
     print_status_streaming(tx)
 
+def tx5():
```
Comment (muratovv) [inline example/python/tx-example.py]: Better to add documentation string with a description of use case. The best choice here use `given, when, then` notation. Same with the next example.
Also, the name looks doubtful. Maybe change it with more suitable and nice for a user?

Comment (kamilsa) [reply example/python/tx-example.py]: It is not a test. We don't have checks on expected behaviour

**Pull request merged**